memory.py

```diff
@@ -73,71 +97,65 @@ def __init__(self, args, capacity):
     self.priority_weight = args.priority_weight  # Initial importance sampling weight β, annealed to 1 over course of training
     self.priority_exponent = args.priority_exponent
     self.t = 0  # Internal episode timestep counter
+    self.reward_scaling = torch.tensor([self.discount ** i for i in range(self.n)], dtype=torch.float32, device=self.device)  # Discount-scaling vector for n future rewards
```

Comment:
Suggestion - rename to `self.n_step_scaling` and comment to `Discount-scaling vector for n-step returns`

Reply:
Done.